bot/modules/nostr/index.js

```diff
@@ -0,0 +1,68 @@
+require('websocket-polyfill');
```

Comment:
why adding this?

Reply:
nostr-tools requires a WebSocket implementation. It does not exist in native node.

https://github.com/nbd-wtf/nostr-tools#interacting-with-a-relay

> To use this on Node.js you first must install websocket-polyfill and import it: